references/plugins/references.go

```diff
@@ -20,18 +20,22 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strconv"
 	"strings"
 
+	"gopkg.in/yaml.v3"
+
```

Comment:
L30 can be removed?

Reply:
I'm using goimports to format the imports. And L30 is added by the goimports tool. I'm afraid if I remove it, it might fail the goimports test.

Reply:
After removing L30, goimport might re-sort the sequence of the imports. Usually, our imports are organized into 3 (4) parts. The official packages, third party packages, (oam-dev packages), kubevela packages.

Reply:
@Nicola115 Once imports are formated, this PR can be merged:)